gui/control-panel.lua

```diff
@@ -127,6 +127,9 @@ local REPEATS = {
     ['orders-sort']={
         desc='Sort manager orders by repeat frequency so one-time orders can be completed.',
         command={'--time', '1', '--timeUnits', 'days', '--command', '[', 'orders', 'sort', ']'}},
+    ['orders-reevaluate']={
+        desc='Invalidates manager orders making it necessary to recheck conditions.',
```

Comment:
I would specifically say "once a month" here. otherwise people might be afraid that their orders will be in a constant reevaluation loop

Reply:
Done